recipe/hardcoded-paths.patch

```diff
@@ -0,0 +1,48 @@
+diff --git gio/gdbusprivate.c gio/gdbusprivate.c
+index 54659c1..7e9d49e 100644
+--- gio/gdbusprivate.c
++++ gio/gdbusprivate.c
+@@ -2099,7 +2099,7 @@ _g_dbus_get_machine_id (GError **error)
+   /* TODO: use PACKAGE_LOCALSTATEDIR ? */
+   ret = NULL;
+   first_error = NULL;
+-  if (!g_file_get_contents ("/var/lib/dbus/machine-id",
++  if (!g_file_get_contents ("@@CONDA_PREFIX@@/var/lib/dbus/machine-id",
```

Comment:
This change and where to look for `dbus`. Though I'm not clear that we are interested in building `dbus` at this time. Though if we do, it will look inside the `conda` prefix, which seems reasonable.